Taskfile.yml

```diff
@@ -275,22 +265,23 @@ tasks:
 
   # set FERRETDB_HANA_URL environment variable to use it
   run-hana:
-    deps: [build-local]
+    desc: "Run FerretDB with `hana` handler"
+    deps: [build-host]
     cmds:
       - >
-        bin/ferretdb-testcover{{exeExt}} -test.coverprofile=cover.txt --
-        --handler=hana
+        bin/ferretdb{{exeExt}} -test.coverprofile=cover.txt --
         --listen-addr=:27017
-        --mode=diff-normal
         --proxy-addr=127.0.0.1:47017
-        --test-records-dir=records
+        --mode=diff-normal
+        --handler=hana
+        --test-records-dir=tmp/records
```

Comment:
Perhaps not related to this change.
```
$ task run-hana
task: [gen-version] go generate -x ./build/version
go run ./generate.go
package.txt: open package.txt: no such file or directory
commit.txt: 404f2d7dd6867bc068e1c3b5a4bb9fbce91354ec
branch.txt: rework-build
version.txt: v0.9.2-49-g404f2d7d
task: [build-host] go test -c -o=bin/ferretdb -trimpath=false -race=true -tags=ferretdb_testcover,ferretdb_debug,ferretdb_tigris,ferretdb_hana -coverpkg=./... ./cmd/ferretdb
task: [run-hana] bin/ferretdb -test.coverprofile=cover.txt -- --listen-addr=:27017 --proxy-addr=127.0.0.1:47017 --mode=diff-normal --handler=hana --test-records-dir=tmp/records

2023-03-10T15:02:47.324+0900    INFO    ferretdb/main.go:216    Starting FerretDB v0.9.2-49-g404f2d7d...        {"version": "v0.9.2-49-g404f2d7d", "commit": "404f2d7dd6867bc068e1c3b5a4bb9fbce91354ec", "branch": "rework-build", "dirty": false, "package": "unknown", "debugBuild": true, "buildEnvironment": {"CGO_CFLAGS":"","CGO_CPPFLAGS":"","CGO_CXXFLAGS":"","CGO_ENABLED":"1","CGO_LDFLAGS":"","GOAMD64":"v1","GOARCH":"amd64","GOOS":"linux","buildtags":"ferretdb_testcover,ferretdb_debug,ferretdb_tigris,ferretdb_hana","compiler":"gc","race":"true"}, "disablePushdown": false, "uuid": "2f96098a-63c3-4240-b419-5b3cc5f6072a"}
2023-03-10T15:02:47.325+0900    FATAL   ferretdb/main.go:321    [hana.go:57 hana.New] HANA instance URL is not provided
github.com/FerretDB/FerretDB/cmd/ferretdb.run
        /home/chi/go/src/github.com/chilagrow/FerretDB/cmd/ferretdb/main.go:321
github.com/FerretDB/FerretDB/cmd/ferretdb.TestCover
        /home/chi/go/src/github.com/chilagrow/FerretDB/cmd/ferretdb/main_test.go:32
testing.tRunner
        /usr/local/go/src/testing/testing.go:1446

```

Reply:
> set FERRETDB_HANA_URL environment variable to use it

We don't put it there because it contains secret credentials

Reply:
Makes sense!